documentation/references.bib

```diff
@@ -0,0 +1,352 @@
+
+@book{ALA01,
+	address = {Reston, VA},
+    author = {American Lifelines Alliance, (ALA)},
```

Comment:
fix indentation throughout the file

Reply:
I'll do my best, BibTeX files tend to be inconsistent